draft-ietf-quic-recovery.md

```diff
@@ -1059,6 +1084,15 @@ A sender that does not use pacing SHOULD reset its congestion window to the
 minimum of the current congestion window and the initial congestion window.
 This recommendation is based on Section 4.1 of {{?RFC5681}}.
 
+## In-Flight Packet Accounting
```

Comment:
I would call this section something else like "Discarding Keys" or "Abandoning Encryption Levels"